edge/pkg/devicetwin/dtmanager/dmiworker.go

```diff
@@ -24,7 +24,7 @@ import (
 	"sync"
 
 	"k8s.io/klog/v2"
-
+	
```

Comment:
Please delete the `Tab` here, just add an empty line, or cilint check will failed @luomengY

Reply:
> Please delete the `Tab` here, just add an empty line, or cilint check will failed @luomengY

ok, It has been removed.